cmd/swarm/main.go

```diff
@@ -574,16 +569,6 @@ func bzzd(ctx *cli.Context) error {
 		stack.Stop()
 	}()
 
-	// Add bootnodes as initial peers.
-	if bzzconfig.BootNodes != "" {
```

Comment:
This `bzzconfig.BootNodes` field is no longer needed I think. You can remove it.